### DISCUSSION TITLE
What about adding Backend features?

### DISCUSSION THREAD

**Discussion by mikolertesx** [nuxt/nuxt#8532] [Ideas]
NextJS has an interesting way to set a backend in the same project, and I think it would be somewhat useful to have here too :).
[Example.](https://github.com/vercel/next.js/blob/canary/examples/api-routes-rest/pages/api/users.js)

Any thoughts?

**Comment by danielroe** [comment]
I think you might be looking for serverMiddleware 🙃

[See docs](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-servermiddleware).

**Reply by mikolertesx** [reply]
Now I feel stupid hahaha.
I did read that but I confused it with middleware.
Even tho, it clearly says "Don't confuse serverMiddleware with middleware".
